docs/design/autoware-interfaces/ad-api/types/autoware_ad_api_msgs/msg/planning_factor.md

```diff
@@ -0,0 +1,42 @@
+# autoware_ad_api_msgs/msg/PlanningFactor
+
+## Definition
+
+```txt
+# constants for type
+uint16 SURROUNDING_OBSTACLE = 1
+uint16 ROUTE_OBSTACLE = 2
+uint16 INTERSECTION = 3
+uint16 CROSSWALK = 4
+uint16 REAR_CHECK = 5
+uint16 USER_DEFINED_DETECTION_AREA = 6
+uint16 NO_STOPPING_AREA = 7
+uint16 STOP_SIGN = 8
+uint16 TRAFFIC_SIGNAL = 9
+uint16 V2I_GATE_CONTROL = 10
+uint16 MERGE = 11
+uint16 SIDEWALK = 12
+uint16 LANE_CHANGE = 13
+uint16 AVOIDANCE1 = 14
+uint16 AVOIDANCE2 = 15
+uint16 DIRECTION_CHANGE = 16
+uint16 EMERGENCY_STOP_OPERATION = 17
+
+# constants for status
+uint16 STOPPING = 1
```

Comment:
Can we precisely judge whether the vehicle is stopping for a factor? I guess it depends on the controller and that the planner can't know that.

Reply:
Removed.